src/programs/sum_all_eer_files/sum_all_eer_files.cpp

```diff
@@ -0,0 +1,215 @@
+#include "../../core/core_headers.h"
+#include <wx/dir.h>
+
+class
+        SumAllEer : public MyApp {
+
+  public:
+    bool DoCalculation( );
+    void DoInteractiveUserInput( );
+
+  private:
+};
+
+IMPLEMENT_APP(SumAllEer)
+
+// override the DoInteractiveUserInput
+
+void SumAllEer::DoInteractiveUserInput( ) {
+
+    int         new_z_size = 1;
+    int         max_threads;
+    std::string output_dark_filename;
+    std::string output_gain_filename;
+    int eer_super_res_factor;
+	int eer_frames_per_image;
+
+    UserInput* my_input = new UserInput("SumAllEerfiles", 1.0);
+
+    std::string output_filename = my_input->GetFilenameFromUser("Output sum file name", "Filename of output image", "output.mrc", false);
+    //bool invert_and_scale         =       my_input->GetYesNoFromUser("Take Reciprocal and Scale?", "If yes, the image will be 1/image and scaled to max density 1.", "YES");
+    bool make_dark_and_gain = my_input->GetYesNoFromUser("Estimate Dark and Gain images", "If yes, a dark and gain image will be estimated", "YES");
+
+    if ( make_dark_and_gain == true ) {
+        output_dark_filename = my_input->GetFilenameFromUser("Output dark file name", "Filename of output dark image", "dark_image.mrc", false);
+        output_gain_filename = my_input->GetFilenameFromUser("Output gain file name", "Filename of output gain image", "gain_image.mrc", false);
+    }
+
+    max_threads = my_input->GetIntFromUser("Max number of threads to use", "maximum number of threads to use for processing.", "1", 1);
+    eer_super_res_factor = my_input-> GetIntFromUser("EER super resolution factor", "super resolution factor for pixels.", "1", 1);
+    eer_frames_per_image = my_input->GetIntFromUser("EER frames per image", "frames per image in eer file type", "0", 0);
+
+    delete my_input;
+
+    my_current_job.Reset(7);
+    my_current_job.ManualSetArguments("tbttiii", output_filename.c_str( ), make_dark_and_gain, output_dark_filename.c_str( ), output_gain_filename.c_str( ), max_threads, eer_super_res_factor, eer_frames_per_image);
```

Comment:
Small formatting detail: please place arguments to ManualSetArguments one/line as in other programs in cisTEM. This is easier to read and modify in the future.

![image](https://user-images.githubusercontent.com/3077528/220993800-4311fd6e-a0ee-4e83-aef0-7b137d619a7d.png)